webviz_ert/assets/ert-style.css

```diff
@@ -62,7 +62,7 @@
   color: black;
 }
 
-.webviz-config-select {
+.ert-select-variable {
```

Comment:
maybe `webviz-ert-select` ... ?

Reply:
ya, naming, one of the hard things in IT : )

I was thinking of following the existing pattern of `ert-` prefixed classes, to kinda signal a namespace, but I don't really mind / care, as I sadly know to little about how to do this properly (namespacing in CSS) to have an opinion

Reply:
I didn't got the full context here and did not know about the ert naming pattern! I'm discarding my comment :+1: